### PR TITLE
DDF-2875 Increase unit test coverage

### PR DESCRIPTION
**Pull request by ahoffer**
#### What does this PR do?
Increases unit test coverage.
#### Who is reviewing it (please choose AT LEAST two reviewers that need to approve the PR before it can get merged)?
@brendan-hofmann @codymacdonald @emmberk @mweser 
#### Select at least one member from relevant component team(s) from below (at least one component team member needs to approve the PR).
@AzGoalie 
#### Choose 2 committers to review/merge the PR (please choose ONLY two committers from 
@clockard
@coyotesqrl
#### How should this be tested? (List steps with links to updated documentation)
mvn install -- build success -- 100% unit test pass and meet code coverage.
#### Any background context you want to provide?
When the approach taken in this PR is approved, I'll do the same thing for ClaimsHandlerManager to bring up the module's unit test coverage some more. 
#### What are the relevant tickets?
[/DDF-2875](https://codice.atlassian.net/browse/DDF-2875/)
#### Checklist:
- [N/A] Documentation Updated
- [N/A] Change Log Updated
- [X] Update / Add Unit Tests
- [N/A] Update / Add Integration Tests

### CODE REVIEW

File: platform/security/sts/security-sts-ldapclaimshandler/src/test/java/ddf/security/sts/claimsHandler/RoleClaimsHandlerTest.java [hunk 1]
```diff
@@ -14,23 +14,126 @@
...
+        assertThat(claim.getValues()
+                .get(0), equalTo(groupName));
+    }
 
-        Assert.assertThat(processedClaims.size(), CoreMatchers.is(equalTo(0)));
+    @Test
+    public void testSupportClaimTypes() {
+        RoleClaimsHandler claimsHandler = new RoleClaimsHandler();
+        List<URI> uris = claimsHandler.getSupportedClaimTypes();
+        assertThat(uris, hasSize(greaterThan(0)));
```
Comment (clockard) [inline platform/security/sts/security-sts-ldapclaimshandler/src/test/java/ddf/security/sts/claimsHandler/RoleClaimsHandlerTest.java]: Shouldn't we know the number of claim types we should have for this test?

File: platform/security/sts/security-sts-ldapclaimshandler/src/test/java/ddf/security/sts/claimsHandler/RoleClaimsHandlerTest.java [hunk 2]
```diff
@@ -14,23 +14,126 @@
...
+        Connection connection;
+        ConnectionEntryReader membershipReader, groupNameReader;
+        LDAPConnectionFactory connectionFactory;
+        LinkedAttribute membershipAttribute, groupNameAttribute;
+        ProcessedClaimCollection processedClaims;
+        RoleClaimsHandler claimsHandler;
+        SearchResultEntry membershipSearchResult, groupNameSearchResult;
+        String groupName = "avengers";
+
+        bindResult = mock(BindResult.class);
```
Comment (clockard) [inline platform/security/sts/security-sts-ldapclaimshandler/src/test/java/ddf/security/sts/claimsHandler/RoleClaimsHandlerTest.java]: Could do the mock inline with the declaration to reduce number of lines for this and the other mocked objects below

**Comment by mweser** [comment]
Passes full swarm hero build.

**Comment by mweser** [comment]
build now

**Comment by clockard** [comment]
build now

**Comment by ahoffer** [comment]
build now

**Pull request merged**